src/server/verification/processor/EnrollmentProcessor.js

```diff
@@ -292,8 +342,8 @@ const enrollmentProcessors = new WeakMap()
 
 export default (storage, log) => {
   if (!enrollmentProcessors.has(storage)) {
-    log = log || logger.child({ from: 'EnrollmentProcessor' })
-    const enrollmentProcessor = new EnrollmentProcessor(Config, storage, AdminWallet, log)
+    const processorLogger = log || logger.child({ from: 'EnrollmentProcessor' })
```

Comment:
actually processor shouldn't encapsulate logger. it should be passed as optional param to each method.
this should be fixed in a separate PR.